data-processing-lib/python/test/data_processing_tests/invoker/python_invoker_test.py

```diff
@@ -0,0 +1,44 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import os
+
+from data_processing.utils import get_logger
+from data_processing.runtime.pure_python import execute_python_transform
+from data_processing.utils import TransformsConfiguration
+
+logger = get_logger(__name__)
+
+
+def test_configuration():
+    """
+    test configuration population
+    :return:
+    """
+    t_configuration = TransformsConfiguration()
+    transforms = t_configuration.get_available_transforms()
+    logger.info(f"available transforms {transforms}")
+    assert len(transforms) == 13
```

Comment:
this magic number, can create issues when we add new transformers. Should we compare number of transformers subdirectories in the `transforms` tree.

Reply:
Yes. Adding transform should change it